modules/priceFloors.js

```diff
@@ -568,8 +568,9 @@ export function handleSetFloorsConfig(config) {
   _floorsConfig = utils.pick(config, [
     'enabled', enabled => enabled !== false, // defaults to true
     'auctionDelay', auctionDelay => auctionDelay || 0,
+    'floorProvider', floorProvider => String(floorProvider) || '',
```

Comment:
I think just leaving this as `floorProvider` is better. If it is not set it will be set as `undefined` which is basically the same thing as `empty string`. Little less code.

Reply:
```
    'auctionDelay', auctionDelay => auctionDelay || 0,
    'floorProvider',
```

Reply:
That makes sense, I wasn't sure how strict to enforce typing for this.